docs/developer-docs/latest/developer-resources/database-apis-reference/graphql-api.md

```diff
@@ -0,0 +1,476 @@
+| `startsWith` | Starts with                      |
+| `endsWith`   | Ends with                        |
+| `and`        | Logical `and`                    |
+| `or`         | Logical `or`                     |
+
+::: request Example query with filters
+
+```graphql
+{
+  documents(filters: { name: { eq: "test" }, or: [{ price: { gt: 10 }}, { title: { startsWith: "Book" }}] }) {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+## Sorting
+
+Queries can accept a `sort` parameter with the following syntax:
+
+- to sort based on a single value: `sort: "value"` 
+- to sort based on multiple values: `sort: ["value1", "value2"]`
+
+The sorting order can be defined with `:asc` (ascending order, default, can be omitted) or `:desc` (for descending order).
+
+::: request Example request: Sorting on title by ascending order
+
+```graphql
+{
+  documents(sort: "title") {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+::: request Example request: Sorting on title by descending order
+
+```graphql
+{
+  documents(sort: "title:desc") {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+::: request Example request: Sorting on title by ascending order, then on price by descending order
+
+```graphql
+{
+  documents(sort: ["title:asc", "price:desc"]) {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+## Pagination
+
+Queries can accept a `pagination` parameter. Results can be paginated either by page or by offset.
+
+:::note
+Pagination methods can not be mixed. Always use either `page` with `pageSize` **or** `start` with `limit`.
+:::
+
+### Pagination by page
+
+<!-- ? are we using a zero-based or a 1-based index? -->
+
+| Parameter              | Description | Default |
+| ---------------------- | ----------- | ------- |
+| `pagination[page]`     | Page number | 1       |
+| `pagination[pageSize]` | Page size   | 100     |
+
+::: request Example query: Pagination by page
+
+```graphql
+{
+  documents(pagination: { page: 1, pageSize: 10 }) {
+    data {
+      id
+    }
+    meta {
+      pagination {
+        page
+        pageSize
+        pageCount
+        total
+      }
+    }
+  }
+}
+```
+
+:::
+
+### Pagination by offset
+
+| Parameter           | Description                  | Default | Maximum |
+| ------------------- | ---------------------------- | ------- | ------- |
+| `pagination[start]` | Start value                  | 0       | -       |
+| `pagination[limit]` | Number of entities to return | -       | 100     
+
+
+::: request Example query: Pagination by offset
+
+```graphql
+{
+  documents(pagination: { start: 20, limit: 30 }) {
+    data {
+      id
+    }
+    meta {
+      pagination {
+        start
+        limit
+      }
+    }
+  }
+}
+```
+
+:::
+
+::: tip
+The default and maximum values for `pagination[limit]` can be [configured in the `./config/api.js`](/developer-docs/latest/setup-deployment-guides/configurations/optional/api.md) file with the `api.rest.defaultLimit` and `api.rest.maxLimit` keys.
```

Comment:
The correct config to tweak is the one located in `./config/plugins.js`.
You then have to play with the `graphql.config.defaultLimit` and `graphql.config.maxLimit` keys.

You can see an example here:
https://github.com/strapi/strapi/blob/5e341674da4f2ef1b99a910e92128674658a80de/examples/getstarted/config/plugins.js#L9-L10